### PR TITLE
added: new variable to be able to disable the default behavior of wrapping draggable elements

### PR DESCRIPTION
**Pull request by webtrix**
Being able to disable this default behavior might come useful for some people (myself included). I'm using your script in an Angular 2 project, and I need to disable this feature to get everything working correctly.

So would be cool if we could add this feature if possible.

### CODE REVIEW

File: dragster.js [hunk 1]
```diff
@@ -201,18 +202,20 @@
          * @return {Array}
          */
         wrapDraggableElements = function (elements) {
-            elements.forEach(function (draggableElement) {
-                var wrapper = createElementWrapper(),
-                    draggableParent = draggableElement.parentNode;
+            if (finalParams.wrapDraggableElements === TRUE) {
```
Comment (sunpietro) [inline dragster.js]: I would do it a different way.

```
wrapDraggableElements = function (elements) {
    if (finalParams.wrapDraggableElements === FALSE) {
        return FALSE;
    }
    
    elements.forEach(function (draggableElement) {
        var wrapper = createElementWrapper(),
            draggableParent = draggableElement.parentNode;

        if (draggableParent.classList.contains(CLASS_DRAGGABLE)) {
            return FALSE;
        }

        draggableParent.insertBefore(wrapper, draggableElement);
        draggableParent.removeChild(draggableElement);
        wrapper.appendChild(draggableElement);
    });
};
```
It avoids the indentation of huge block of code.

**Comment by webtrix** [comment]
Made the requested changes.

File: dragster.js [hunk 2]
```diff
@@ -201,6 +202,17 @@
          * @return {Array}
          */
         wrapDraggableElements = function (elements) {
+            if (finalParams.wrapDraggableElements === TRUE) {
```
Comment (sunpietro) [inline dragster.js]: `finalParams.wrapDraggableElements === FALSE`

Comment (webtrix) [inline dragster.js]: OMG .. my bad .. sry

**Comment by webtrix** [comment]
Fixed

**Comment by sunpietro** [comment]
@webtrix can you squash the commits into the one named: `added: new variable to be able to disable the default behavior of wrapping draggable elements`?

**Comment by webtrix** [comment]
Took me some searching but this OK?

**Pull request merged**